utils/swaggerDefinition.js

```diff
@@ -140,6 +140,155 @@ const swaggerOptions = {
             }
           }
         },
+        contributions: {
+          type: 'object',
+          properties: {
+            noteworthy: {
+              type: 'array',
+              items: {
+                task: {
+                  type: 'object',
+                  properties: {
+                    title: {
+                      type: 'string'
+                    },
+                    purpose: {
+                      type: 'string'
+                    },
+                    featureUrl: {
+                      type: 'string'
+                    },
+                    endsOn: {
+                      type: 'string'
+                    },
+                    startedOn: {
+                      type: 'string'
+                    },
+                    dependsOn: {
+                      type: 'array',
+                      items: {
+                        taskid: {
+                          type: 'string'
+                        }
+                      }
+                    },
+                    participants: {
+                      type: 'array',
+                      items: {
+                        userid: {
+                          type: 'string'
+                        }
+                      }
+                    },
+                    isNoteworthy: {
+                      type: 'boolean'
+                    }
+                  }
+                },
+                prList: {
+                  type: 'array',
+                  items: {
+                    pullRequests: {
```

Comment:
This seems to be redundant and used at multiple places. Can we add a reference for this to avoid redundant declarations?

Reply:
PR object result is different at both the places